src/components/forms/CloudInitForm.tsx

```diff
@@ -29,12 +32,76 @@ interface Props {
 }
 
 const CloudInitForm: FC<Props> = ({ formik }) => {
-  const codeRenderer = (value?: unknown) =>
-    value === "-" || value === undefined ? (
-      ""
-    ) : (
-      <CloudInitConfig config={value as string} />
-    );
+  const getCloudInitRow = (label: string, name: string, value?: string) => {
```

Comment:
nice refactor, the cloud init form looks much better now. Shall we move `getCloudInitRow` into it's own component?

Reply:
As it is only relevant to this component, and we are accessing values from the component, moving it would increase the interface, probably better to keep it local.